docs/install.md

```diff
@@ -1,77 +1,197 @@
-
-#### GCS objects
-If you have raw disk images from physical machines or otherwise that you want to process in the cloud, you can process them directly from GCS.  This can be used when the images do not originate from VMs/containers that are running in the cloud.  You can potentially convert raw images to Persistent Disks, but that requires processing the image first, and so using GCS is recommended in this case.  Processing Evidence directly from GCS requires that you use [GCS FUSE](https://cloud.google.com/storage/docs/gcs-fuse) and mount your bucket at a common path on each worker node.  Once your GCS bucket is mounted, you can process these images as 'rawdisk' Evidence.
+# High-Level Setup
+
+Turbinia can be run either in the Google Cloud, or on local machines. If you run
+Turbinia on local machines, it will still use [Cloud
+Pub/Sub](https://cloud.google.com/pubsub) and [Cloud
+Functions](https://cloud.google.com/functions) for the client to talk to the
+server, and for the server to talk to the worker nodes.
+
+## Local Setup
+
+Turbinia requires all worker nodes to have direct access to all Evidence data.
+The easiest way to set this up on local machines is to have a NFS or SAN mounted
+on a common path on each worker. All output should also be written to the common
+directory so that when Evidence is added back into Turbinia that the other
+worker nodes can process it. Turbinia can also write output to GCS even when
+running locally (set the `GCS_OUTPUT_PATH` variable in the config).
+
+## Google Cloud Platform (GCP) Setup
+
+Turbinia can read Evidence from either cloud Persistent Disks, or from GCS
+objects. Turbinia can also write output to GCS (set the `GCS_OUTPUT_PATH`
+variable in the config). Note that you can operate multiple Turbinia instances
+within the same GCP Project as long as your careful to make sure your config
+(Pub/Sub topics/subscriptions, output paths, etc) doesn't overlap.
+
+### Persistent Disks
+
+Persistent disks should be the default when processing disks that come from the
+Cloud. The account you run Turbinia as must have access to the Persistent Disks
+that you want to process. If you add a GoogleCloudDisk Evidence type, the worker
+node will attach the disk automatically before it runs its tasks.
+
+### GCS Objects
+
+If you have raw disk images from physical machines or otherwise that you want to
+process in the cloud, you can process them directly from GCS. This can be used
+when the images do not originate from VMs/containers that are running in the
+cloud. You can potentially convert raw images to Persistent Disks, but that
+requires processing the image first, and so using GCS is recommended in this
+case. Processing Evidence directly from GCS requires that you use [GCS
+FUSE](https://cloud.google.com/storage/docs/gcs-fuse) and mount your bucket at a
+common path on each worker node. Once your GCS bucket is mounted, you can
+process these images as 'rawdisk' Evidence.
```

Comment:
So, this section was true when it was written, but we've since found that processing images directly from GCS does not work well.  Feel free to leave it for now, but we should make sure to update it to show how to use GoogleCloudDiskRawEmbedded disks instead.

Reply:
Added a todo line

Reply:
done